Tests/MongoSwiftTests/CodecTests.swift

```diff
@@ -484,17 +484,20 @@ final class CodecTests: XCTestCase {
         """.data(using: .utf8)!
 
         let expected: Document = [
```

Comment:
I reordered this to make it match the order of the JSON we parse, since before it was just hard-coded in the expected order